bin/servicerouter.py

```diff
@@ -1,60 +1,118 @@
 #!/usr/bin/env python2
 
-"""Updates haproxy config based on marathon.
+"""Overview:
+  Servicerouter is a replacement for the haproxy-marathon-bridge.
+  It reads Marathon task information and generates haproxy configuration.
+
+  Service configuration lives in Marathon via environment variables.
+  The servicerouter just needs to know where to find Marathon.
+
+  Every service port in Marathon can be configured independently.
+
 
 Features:
   - Virtual Host aliases for services
   - Soft restart of haproxy
   - SSL Termination
-  - (Optional): real-time update from marathon events
-
-Usage:
-./servicerouter.py --listening http://192.168.0.1:4002 -m http://marathon1
-./servicerouter.py -m http://marathon1 -m http://marathon2:8080 -m http://marathon3:8080
+  - (Optional): real-time update from Marathon events
 
-HA Usage:
-  TODO (Run on multiple hosts, send traffic to all)
 
 Configuration:
   Service configuration lives in marathon via environment variables.
   The servicerouter just needs to know where to find marathon.
   To run in listening mode you must also specify the address + port at
-  which the servicerouter can be reached by marathon.
 
-  Every service port in marathon can be configured independently.
 
-  Environment Variables:
-    HAPROXY_GROUP
-      Only servicerouter instances which are members of the given group will
-      point to the service. Service routers with the gorup '*' will collect all
-      groups.
+Usage:
+  $ servicerouter.py --marathon http://marathon1:8080 --haproxy-config /etc/haproxy/haproxy.cfg
+
+  The user servicerouter is executed as must have the right to reload haproxy.
 
-    HAPROXY_{n}_VHOST
-      HTTP Virtual Host proxy hostname to catch
-      Ex: HAPROXY_0_VHOST = 'marathon.mesosphere.com'
 
-    HAPROXY_{n}_STICKY
-      Use sticky request routing for the service
-      Ex: HAPROXY_0_STICKY = true
+Environment Variables:
+  HAPROXY_GROUP
+  Only servicerouter instances which are members of the given group will
+  point to the service. Service routers with the group '*' will collect
+      all groups.
```

Comment:
What about something like:

"only apps which are members of the given group will be used when generating the configuration file. If the group '*' is specified, then all the apps will be used." ?